test/schedule/MongoSchedule.integration.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { getConnection } from 'typeorm';
+
+import { clear, MomoJob, MongoSchedule } from '../../src';
+import { connectionName } from '../../src/connect';
+import { JobRepository } from '../../src/repository/JobRepository';
+import { initLoggingForTests } from '../utils/logging';
+import { JobEntity } from '../../src/repository/JobEntity';
+import { withDefaults } from '../../src/job/withDefaults';
+
+describe('MongoSchedule', () => {
+  const job: MomoJob = {
+    name: 'test job',
+    interval: 'one minute',
+    handler: jest.fn(),
+  };
+
+  let mongo: MongoMemoryServer;
+  let jobRepository: JobRepository;
+  let mongoSchedule: MongoSchedule;
+
+  beforeAll(async () => {
+    mongo = await MongoMemoryServer.create();
+    mongoSchedule = await MongoSchedule.connect({ url: await mongo.getUri() });
+    jobRepository = getConnection(connectionName).getCustomRepository(JobRepository);
+
+    initLoggingForTests(mongoSchedule);
+  });
+
+  beforeEach(async () => {
+    mongoSchedule.cancel();
+    await clear();
+  });
+
+  afterAll(async () => {
+    await mongoSchedule.disconnect();
+    await mongo.stop();
+  });
+
+  it('saves job with defaults', async () => {
+    await mongoSchedule.define(job);
+
+    expect(await mongoSchedule.get(job.name)).toEqual({
+      name: job.name,
+      interval: job.interval,
+      concurrency: 1,
+      maxRunning: 0,
```

Comment:
since we do not have a real `disabled` option users might consider `maxRunning=0` like disabled. Reading the docs helps of course, but from other libraries I would expect a negative value to mean "no limit". Zero could be a bit ambiguous here.

Reply:
We can discuss to change this (in a separate PR maybe?)